### ISSUE TITLE
Adaptive Server Connection Failed on Windows pymssql >=2.2.8

### ISSUE THREAD

**Issue by tbronson** [pymssql/pymssql#860]
SQL Server ver. 12.0.2000.8

with pymssql versions <= 2.2.7 this works fine
```
import pymssql
pymssql.connect(server="localhost")
```
using the local windows account authentication

however on pymssql versions >2.2.8 I get this instead:
```
Traceback (most recent call last):
  File "src\\pymssql\\_pymssql.pyx", line 647, in pymssql._pymssql.connect
  File "src\\pymssql\\_mssql.pyx", line 2109, in pymssql._mssql.connect
  File "src\\pymssql\\_mssql.pyx", line 701, in pymssql._mssql.MSSQLConnection.__init__
  File "src\\pymssql\\_mssql.pyx", line 1818, in pymssql._mssql.maybe_raise_MSSQLDatabaseException
  File "src\\pymssql\\_mssql.pyx", line 1835, in pymssql._mssql.raise_MSSQLDatabaseException
pymssql._mssql.MSSQLDatabaseException: (20002, b'DB-Lib error message 20002, severity 9:\nAdaptive Server connection failed (localhost)\nDB-Lib error message 20002, severity 9:\nAdaptive Server connection failed (localhost)\n')

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "src\\pymssql\\_pymssql.pyx", line 653, in pymssql._pymssql.connect
pymssql._pymssql.OperationalError: (20002, b'DB-Lib error message 20002, severity 9:\nAdaptive Server connection failed (localhost)\nDB-Lib error message 20002, severity 9:\nAdaptive Server connection failed (localhost)\n')
```

**Comment by tbronson** [comment]
After some searching I solved the issue by setting tds_version to '7.0' as follows:

`pymssql.connect(server="localhost", tds_version="7.0")`

and everything works as normal again.

**Comment by Kyrie666** [comment]
nice
